pysat/tests/classes/cls_instrument_library.py

```diff
@@ -90,11 +91,19 @@ def load_and_set_strict_time_flag(test_inst, date, raise_error=False,
     clean_off : bool
         Turn off the clean method when re-loading data and testing the
         strict time flag (default=True)
+    concat : bool
+        If True, load multiple days to concat. If False, load single day.
+        (default=False)
 
     """
 
+    kwargs = {'use_header': True}
```

Comment:
We don't need this anymore. Recommend something like `kwargs = {'date': date}` and adjusting the load calls below.